ruby/ext/google/protobuf_c/repeated_field.c

```diff
@@ -199,13 +200,13 @@ void* RepeatedField_index_native(VALUE _self, int index) {
  * Removes the last element and returns it. Throws an exception if the repeated
  * field is empty.
  */
-VALUE RepeatedField_pop(VALUE _self) {
+VALUE RepeatedField_pop_internal(VALUE _self) {
```

Comment:
Sounds good; change method name in doc comment above too? Also the name could be more descriptive -- `pop_one` maybe?